add.go

```diff
@@ -1,8 +1,71 @@
 package main
 
-import "fmt"
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"math/rand"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+	"go.mongodb.org/mongo-driver/mongo"
+)
 
 // Add : Entry point for adding a new document to the database
-func Add() {
-	fmt.Println("Add")
+func Add(ctx context.Context, client *mongo.Client, scanner *bufio.Scanner) error {
+	dummyBson := bson.M{
+		"cast":primitive.A{},
+		"country":primitive.A{}, 
+		"date_added": "", 
+		"description": "When a prison ship crash unleashes hundreds of Decepticons on Earth, Bumblebee leads a new Autobot force to protect humankind.",
+		"director": primitive.A{},
+		"duration": "", 
+		"listed_in": primitive.A{}, 
+		"rating": "", 
+		"release_year": 0.0,
+		"title": "",
+		"type": "",
+	}
+
+	var randomId int
+	checkIfRepeated := true
+	for checkIfRepeated {
+		randomId = rand.Intn(99999999)
+		filter := bson.D{primitive.E{Key: "show_id", Value: randomId}}
+		var checkResult bson.M
+		err := GetTitlesColl(client).FindOne(ctx, filter).Decode(&checkResult)
+		if err != nil && err == mongo.ErrNoDocuments {
+			checkIfRepeated = false
```

Comment:
Igual y el bool `checkIfRepeated` sale sobrando, no? Podrías hacer un loop infinito (con `for {`), y ahí hacer un break